src/platform/testing/e2e/cypress/support/commands/formHelpers.js

```diff
@@ -37,12 +37,18 @@ Cypress.Commands.add('selectRadio', (fieldName, value) => {
     });
   }
 });
+/**
+ * Works with Date widget. And va-date and va-memorable-date web components
+ */
 Cypress.Commands.add('fillDate', (fieldName, dateString) => {
   // Split the date & remove leading zeros
   const date = dateString
     .split('-')
     .map(number => parseInt(number, 10).toString());
   cy.document().then(doc => {
+    const vaMemorableDate = doc.querySelector(
+      `va-memorable-date[name="${fieldName}"]`,
+    );
     const vaDate = doc.querySelector(`va-date[name="${fieldName}"]`);
```

Comment:
is `vaDate` still needed here?

Reply:
Yes, `fillDate` will work with a React Date component, `va-date`, and `va-memorable-date` web components.